src/appleseed/foundation/math/matrix.h

```diff
@@ -45,7 +45,18 @@
 // Imath headers.
 #ifdef APPLESEED_ENABLE_IMATH_INTEROP
 #include "foundation/platform/_beginexrheaders.h"
+
```

Comment:
Same here. Please, remove newly added pragmas for all `src/appleseed` headers below.